jenkinsapi/build.py

```diff
@@ -92,12 +92,11 @@ def get_revision_branch(self):
         return getattr(self, '_get_%s_rev_branch' % vcs, lambda: None)()
 
     def get_params(self):
-        result = {}
-        parameters = self._data.get('actions', {})[0].get('parameters', {})
-        for pair in parameters:
-            key = pair['name']
-            value = pair['value']
-            result[key] = value
+        """
+        Return a dictionary of params names and their values
+        """
+        parameters = self._data.get('actions', {})[0].get('parameters', )
```

Comment:
If there is no `actions` result of `get` will be empty dict and `{}[0]` will break code...